src/editor/config/config.ts

```diff
@@ -1,4 +1,199 @@
+  canvasCss?: string;
+
+  // Default command
+  defaultCommand?: string;
+
+  // Show a toolbar when the component is selected
+  showToolbar?: boolean;
+
+  // Allow script tag importing
+  allowScripts?: boolean;
+
+  // If true render a select of available devices
+  showDevices?: boolean;
+
+  // When enabled, on device change media rules won't be created
+  devicePreviewMode?: boolean;
+
+  // THe condition to use for media queries, eg. 'max-width'
+  // Comes handy for mobile-first cases
+  mediaCondition?: string;
+
+  // Starting tag for variable inside scripts in Components
+  tagVarStart?: string;
+
+  // Ending tag for variable inside scripts in Components
+  tagVarEnd?: string;
+
+  // When false, removes empty text nodes when parsed, unless they contain a space
+  keepEmptyTextNodes?: boolean;
+
+  // Return JS of components inside HTML from 'editor.getHtml()'
+  jsInHtml?: boolean;
+
+  // Enable native HTML5 drag and drop
+  nativeDnD?: boolean;
+
+  // Enable multiple selection
+  multipleSelection?: boolean;
+
+  // Show the wrapper component in the final code, eg. in editor.getHtml()
+  exportWrapper?: boolean;
+
+  // The wrapper, if visible, will be shown as a `<body>`
+  wrappesIsBody?: boolean;
+
+  // Usually when you update the `style` of the component this changes the
+  // element's `style` attribute. Unfortunately, inline styling doesn't allow
+  // use of media queries (@media) or even pseudo selectors (eg. :hover).
+  // When `avoidInlineStyle` is true all styles are inserted inside the css rule
+  avoidInlineStyle?: boolean;
+
+  // Avoid default properties from storable JSON data, like `components` and `styles`.
+  // With this option enabled your data will be smaller (usefull if need to
+  // save some storage space)
+  avoidDefaults?: boolean;
+
+  // (experimental)
+  // The structure of components is always on the screen but it's not the same
+  // for style rules. When you delete a component you might leave a lot of styles
+  // which will never be used again, therefore they might be removed.
+  // With this option set to true, styles not used from the CSS generator (so in
+  // any case where `CssGenerator.build` is used) will be removed automatically.
+  // But be careful, not always leaving the style not used mean you wouldn't
+  // use it later, but this option comes really handy when deal with big templates.
+  clearStyles?: boolean;
+
+  // Specify the global drag mode of components. By default, components are moved
+  // following the HTML flow. Two other options are available:
+  // 'absolute' - Move components absolutely (design tools way)
+  // 'translate' - Use translate CSS from transform property
+  // To get more about this feature read: https://github.com/artf/grapesjs/issues/1936
+  dragMode?: boolean;
+
+  // Dom element
+  el?: Element | string,
+
+  // Configurations for Undo Manager
+  //TODO: convert configs to ts
+  undoManager?: any,
+
+  //Configurations for Asset Manager
```

Comment:
Missing space character after `//` in this line and a number of following lines.